tests/integration/test_gitlab.py

```diff
@@ -176,23 +180,33 @@ def test_issue_info(self):
         assert issue_info.description.startswith("This is testing issue")
 
     def test_create_issue(self):
-        issue = self.project.create_issue(
-            title="Issue 2", description="Description for issue 2"
-        )
-        assert issue.title == "Issue 2"
-        assert issue.description == "Description for issue 2"
+        """
+        see class comment in case of fail
+        """
+        issue_title = f"New Issue {self.random_str}"
+        issue_desc = f"Description for issue {self.random_str}"
+        issue = self.project.create_issue(title=issue_title, description=issue_desc)
+        assert issue.title == issue_title
+        assert issue.description == issue_desc
 
     def test_close_issue(self):
-        issue_for_closing = self.project.get_issue_info(issue_id=2)
+        """
+        see class comment in case of fail
+        """
+        issue = self.project.create_issue(
```

Comment:
I don't like creating an issue in this test, but we can leave it here so the regeneration is easier.